release/build_release.sh

```diff
@@ -106,6 +106,35 @@ function cherry_pick_commits {
 	git push origin -q -u "${LIFERAY_RELEASE_GIT_REF}"
 }
 
+function create_branch {
+	local original_branch_name=""
+
+	if [ "${LIFERAY_RELEASE_SOFT}" == "true" ]
+	then
+		original_branch_name="${LIFERAY_RELEASE_GIT_PREV_REF}"
+	else
+		original_branch_name="master"
+	fi
+
+	local last_commit_sha=$(curl -s -H "Authorization: token ${LIFERAY_RELEASE_GIT_TOKEN_SECRET}" https://api.github.com/repos/liferay/liferay-portal-ee/git/refs/heads/"${original_branch_name}" | jq -r '.object.sha')
+
+	local commit_interval_json=$(jq -n --arg ref "refs/heads/$LIFERAY_RELEASE_GIT_REF" --arg sha "$last_commit_sha" '{ref: $ref, sha: $sha}')
+
+	local http_response_code=$(curl -o /dev/null -w "%{response_code}" -s -X POST -H "Authorization: token ${LIFERAY_RELEASE_GIT_TOKEN_SECRET}" -d "${commit_interval_json}" https://api.github.com/repos/liferay/liferay-portal-ee/git/refs)
```

Comment:
@kiwm, please check if we can use `lc_curl` instead to keep code consistency

Reply:
I believe that doesn't make sense for those cases.